### PR TITLE
[TACHYON-369] Refactor method names in MasterInfo.java to be consistent 

### PR DESCRIPTION
**Pull request by apc999**
with other parts of code

### CODE REVIEW

File: core/src/main/java/tachyon/master/MasterInfo.java [hunk 1]
```diff
@@ -2133,6 +2135,21 @@ public boolean rename(TachyonURI srcPath, TachyonURI dstPath) throws FileDoesNot
...
+   * Rename a file. This method is supposed to be called in unittest only.
+   *
+   * @param fileId The id of the file to rename
+   * @param dstPath The new path of the file
+   * @param opTimeMs The time of the rename operation, in milliseconds
+   * @return true if the rename succeeded, false otherwise
+   * @throws FileDoesNotExistException If the id doesn't point to an inode
+   * @throws InvalidPathException if the source path is a prefix of the destination
+   */
+  public boolean rename(int fileId, TachyonURI dstPath, long opTimeMs)
```
Comment (haoyuan) [inline core/src/main/java/tachyon/master/MasterInfo.java]: If this is called by unit test only, make it "package" scope is fine.

File: core/src/test/java/tachyon/master/MasterInfoTest.java [hunk 2]
```diff
@@ -584,7 +584,7 @@ public void lastModificationTimeRenameTest() throws InvalidPathException,
         mMasterInfo.createFile(new TachyonURI("/testFolder/testFile1"),
             Constants.DEFAULT_BLOCK_SIZE_BYTE);
     long opTimeMs = System.currentTimeMillis();
-    Assert.assertTrue(mMasterInfo._rename(fileId, new TachyonURI("/testFolder/testFile2"),
+    Assert.assertTrue(mMasterInfo.rename(fileId, new TachyonURI("/testFolder/testFile2"),
```
Comment (haoyuan) [inline core/src/test/java/tachyon/master/MasterInfoTest.java]: why not call mMasterInfo.renameInternal directly?

File: core/src/main/java/tachyon/master/MasterInfo.java (hunk 1 above)
Comment (apc999) [inline core/src/main/java/tachyon/master/MasterInfo.java]: This wrapper method is removed, as the unittest now directly calls renameInternal now.

File: core/src/test/java/tachyon/master/MasterInfoTest.java (hunk 2 above)
Comment (apc999) [inline core/src/test/java/tachyon/master/MasterInfoTest.java]: Done.

**Pull request merged**